Source/Csla.Blazor/State/StateManager.cs

```diff
@@ -57,12 +56,12 @@ private async Task GetState(TimeSpan timeout)
     /// at which you know the user is navigating to another
     /// page.
     /// </remarks>
-    public async Task SaveState()
+    public async Task SaveState(TimeSpan timeout)
```

Comment:
To make your life easier with the comments here I suggest using `/// <inheritdoc />` [Docs](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#inheritdoc)